resume.md

```diff
@@ -1,49 +1,65 @@
-# FIRST LAST NAME
+# Armen Phelps
 
-(123) 456 - 7890 | email@gmail.com
+(510) 846 - 6136 | armenphelps204@berkeley.edu
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+https://www.linkedin.com/in/armenphelps2022/
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+University of California, Berkeley | B.A. Computer Science, B.A. Applied Math (Double Major) | May 2022
+Coursework: Structure & Interpretation of Computer Programs; Data Structures; Computer Architecture (Machine Structures); Computer Security; Efficient Algorithms & Intractable Problems; Designing Information Devices & Systems; Abstract Algebra; Discrete Math & Probability Theory; Linear Algebra & Differential Equations; Multivariable Calculus
```

Comment:
use the common names of classes.  Recruters won't know what these are
* ~~Structure & Interpretation of Computer Program~~
* ~~Efficient Algorithms & Intractable Problems~~ --> Efficient Algorithms